### PR TITLE
Search hotkey, button ui tweaks, adding clear to search

### PR DESCRIPTION
**Pull request by LouieBloo**

### CODE REVIEW

File: src/app/components/card-list/card-list.component.html [hunk 1]
```diff
@@ -1,7 +1,10 @@
 <div>
     <div class="flex justify-center mb-2">
         <button class="hidden" onclick="searchModal.showModal()" id="toggleModal">hidden</button>
-        <button class="btn btn-sm w-full btn-outline text-white" (click)="openSearchModal()" id="luke">Search Cards (ctrl + i)</button>
+        <div class="join flex w-full">
+            <button class="btn btn-sm btn-outline text-white join-item rounded-none flex-1" (click)="openSearchModal()" id="luke">Search Cards (ctrl + q)</button>
```
Comment (domleggio) [inline src/app/components/card-list/card-list.component.html]: id is "luke". Do we need an id?

File: src/app/components/card-list/card-list.component.ts [hunk 2]
```diff
@@ -184,4 +184,8 @@ export class CardListComponent {
       closeModalButton.click();
     }
   }
+
+  clearSearchHistory = ()=>{
```
Comment (domleggio) [inline src/app/components/card-list/card-list.component.ts:188]: do you like this style for functions? you can just write the function without the `= ()`

**Review by domleggio — APPROVED** [review]

File: src/app/components/card-list/card-list.component.html (hunk 1 above)
Comment (LouieBloo) [reply src/app/components/card-list/card-list.component.html]: fixed

File: src/app/components/card-list/card-list.component.ts (hunk 2 above)
Comment (LouieBloo) [reply src/app/components/card-list/card-list.component.ts:188]: there is a difference, doesnt matter in this case but if you do => it preserves the scope

**Pull request merged**